.github/workflows/ci.yml

```diff
@@ -0,0 +1,214 @@
+name: CI
+on: [push]
+
+jobs:
+  ci-image:
+    name: Build CI image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Remove dockerignore
+        run: |
+          rm .dockerignore
+      - name: Build and push Docker images
+        uses: whoan/docker-build-with-cache-action@v5
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          image_name: "dependabot/dependabot-core-ci"
+          dockerfile: "Dockerfile.ci"
+          image_tag: ${{ github.sha }}
+  rubocop:
+    name: Rubocop
+    runs-on: ubuntu-latest
+    container: dependabot/dependabot-core-ci:${{ github.sha }}
+    needs: ci-image
+    steps:
+      - name: Run Rubocop linting
+        run: |
+          cd /home/dependabot/dependabot-core && rake ci:rubocop
```

Comment:
```suggestion
        run: rake ci:rubocop
```

Reply:
Head spin rt round. Is there a main talk/chat group?

Reply:
That doesn't work unfortunately: https://github.com/jurre/dependabot-core/runs/906886314?check_suite_focus=true